client/src/pages/Room/Room.tsx

```diff
@@ -93,9 +99,11 @@ const Room: React.FC = () => {
   // submit message
   const onMessageSubmit = useCallback(
     (event: React.FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
       sendMessage({
         variables: {
           content: (event.target as any).message.value,
+          mentions: mentions.map(m => m.id),
```

Comment:
mentions have to array of IDs thats why i'm maping out the id from {display, id} in react-mention's MentionItm interface